taxi/forms.py

```diff
@@ -0,0 +1,44 @@
+import re
+
+from django.contrib.auth.forms import UserCreationForm
+from django import forms
+
+from taxi.models import Driver, Car
+
+
+class DriverCreateForm(UserCreationForm):
+    class Meta(UserCreationForm.Meta):
+        model = Driver
+        fields = UserCreationForm.Meta.fields + (
+            "username",
+            "first_name",
+            "last_name",
+            "license_number",
+            "email",)
```

Comment:
The `fields` attribute in `DriverCreateForm` should not include `username` since it is already part of `UserCreationForm.Meta.fields`. Including it again is redundant and could lead to unexpected behavior.